common/common.cc

```
@@ -252,8 +252,11 @@ void appendFilesInDir(string_view basePath, string_view path, const sorbet::Unor
             if (!recursive || strcmp(entry->d_name, ".") == 0 || strcmp(entry->d_name, "..") == 0) {
                 continue;
             }
-            appendFilesInDir(basePath, fullPath, extensions, recursive, result, absoluteIgnorePatterns,
-                             relativeIgnorePatterns);
+            try {
+                appendFilesInDir(basePath, fullPath, extensions, recursive, result, absoluteIgnorePatterns,
+                                 relativeIgnorePatterns);
+            } catch (sorbet::FileNotFoundException) {
+            }
```

can we report an error here rather than swallow the exception?